pages/index.vue

```diff
@@ -4,6 +4,15 @@
       title="IPFS Powers the Distributed Web"
       description="The InterPlanetary File System is a peer-to-peer hypermedia protocol designed to preserve and grow humanity's knowledge by making the web upgradeable, resilient, and more open."
     />
+    <Hero background="star-aurora.jpg" overlay="constellations-hero.svg" :tall="true">
```

Comment:
```suggestion
    <Hero background="star-aurora.jpg" overlay="constellations-hero.svg" tall>
```
nit but should be the same as `true`